fe/fe-core/src/test/java/com/starrocks/http/BadHttpRequestTest.java

```diff
@@ -60,6 +60,21 @@ public void testStreamLoadWithNormalRequest() throws IOException {
         Assert.assertEquals(200, response.code());
     }
 
+    @Test
+    public void testInvalidAuthorityRequest() throws IOException {
+        RequestBody body = RequestBody.create(JSON, "{}");
+        Request.Builder requestBuilder = new Request.Builder()
+                .url(String.format(STREAM_LOAD_URL_FORMAT, DB_NAME, TABLE_NAME))
+                .put(body)
+                .addHeader("Authorization", "Basic xxx")
+                .addHeader("columns", genColumnNameList(Config.http_max_header_size))
+                .addHeader("label", UUID.randomUUID().toString())
+                .addHeader("Expect", "100-continue");
+        Request request = requestBuilder.build();
+        Response response = networkClient.newCall(request).execute();
+        Assert.assertEquals(200, response.code());
```

Comment:
need to fix this UT, and should expect HTTP 401 error for auth fail.

Reply:
@kevincai hello kevin, i had fixed the ut issue, please help me to review the code again~thank you~